### ISSUE TITLE
Let the user adjust the position of any keyframe by dragging its marker on the timeline

### ISSUE THREAD

**Issue by raindropsfromsky** [OpenShot/openshot-qt#5435]
Many apps (3D modelers and video editors) allow ther user to drag keyframes along the timeline to change its position.
But Openshot only allows the user to remove the keyframe.
So if he only wanted to change the timing of that keyframe, he has to remove the keyframe and insert another one at the new time.

Instead, if he can simply drag the keyframe marker on the timeline itself, it would be much more intuitive.

If accidental dragging is a concern, let there be a modifier key (e.g. CTRL+Drag) to prevent accidental dragging of the keyframe.

**Comment by leodf1** [comment]
Agreed. This appears to have been an intended function, since the cursor already changes to the finger when over a KF. Don't see accidental dragging as a concern....

**Comment by ivandortulov** [comment]
Will this get implemented?

**Comment by Sir-Kam** [comment]
> Will this get implemented?

It's a request that goes way back. It was discussed here #689 and still has yet to be added.

**Comment by Colorjet3** [comment]
It is in the queue but I couldn't give you an ETA.
